src/articulation.ts

```diff
@@ -318,15 +314,13 @@ export class Articulation extends Modifier {
     // Use type as glyph code, if not defined as articulation code
     if (!this.articulation) {
       this.articulation = { code: this.type, betweenLines: false };
-      if (this.type.endsWith('Above')) this.position = ABOVE;
-      if (this.type.endsWith('Below')) this.position = BELOW;
     }
     const code =
-      (this.position === ABOVE ? this.articulation.aboveCode : this.articulation.belowCode) || this.articulation.code;
-    this.glyph = new Glyph(code ?? '', this.renderOptions.fontScale);
-    defined(this.glyph, 'ArgumentError', `Articulation not found: ${this.type}`);
-
-    this.setWidth(defined(this.glyph.getMetrics().width));
+      (this.position === ABOVE ? this.articulation.aboveCode : this.articulation.belowCode) ||
+      this.articulation.code ||
+      '\u0000';
```

Comment:
I don't understand this part.

Is it doing some sanity check with fallback? Why is the fallback the unicode codepoint 0? Should it be an empty string instead?

Reply:
I used 0 becuase this code on some fonts displays a square or 'NOGLYPH' showing that there is a problem.